classes/patreon_metabox.php

```diff
@@ -67,12 +76,8 @@ function patreon_plugin_save_post_class_meta( $post_id, $post ) {
 			$new_patreon_level = 0;
 		} else {
 
-			if($post->post_type == 'patreon-content') {
-				$new_patreon_level = 9999;
-			} else {
-				$new_patreon_level = 0;
-			}
-
+			$new_patreon_level = 0;
+			
```

Comment:
isn't this just
```
if(isset( $_POST['patreon-level']) && is_numeric($_POST['patreon-level'])) {
    $new_patreon_level = $_POST['patreon-level'];
} else {
    $new_patreon_level = 0;
}
```

Reply:
Yes, it can be concatenated to that simpler version:

https://github.com/codebard/patreon-wordpress/commit/a3d4c9e3f8bb15a58bcd00b3c58cd99a55947d02